spark/src/main/resources/error/delta-error-classes.json

```diff
@@ -720,6 +720,45 @@
     ],
     "sqlState" : "0AKDE"
   },
+  "DELTA_FEATURE_DROP_CONFLICT_REVALIDATION_FAIL" : {
+    "message" : [
+      "Cannot drop feature because a concurrent transaction modified the table.",
+      "Please try the operation again.",
+      "<concurrentCommit>"
+    ],
+    "sqlState" : "0AKDE"
+  },
+  "DELTA_FEATURE_DROP_LEGACY_FEATURE" : {
+    "message" : [
+      "Cannot drop <feature> because it is implicitly supported by the table protocol version."
+    ],
+    "sqlState" : "0AKDE"
+  },
+  "DELTA_FEATURE_DROP_NONREMOVABLE_FEATURE" : {
+    "message" : [
+      "Cannot drop <feature> because removing this feature is not supported."
+    ],
+    "sqlState" : "0AKDE"
+  },
+  "DELTA_FEATURE_DROP_UNSUPPORTED_CLIENT_FEATURE" : {
+    "message" : [
+      "Cannot drop <feature> because it is not supported by this Delta version.",
+      "Consider using Delta with a higher version."
+    ],
+    "sqlState" : "0AKDE"
+  },
+  "DELTA_FEATURE_DROP_UNSUPPORTED_FEATURE_TYPE" : {
```

Comment:
What is a "feature type"? What is the feature type of this feature that is not supported? read/write/strange/charm/top/bottom? :) AFAICT this is only ever used to say that we can't drop reader features, and there's no equivalent error for writer features. So let's just make this error say that.